src/compiler/types.ts

```diff
@@ -3731,7 +3731,7 @@ namespace ts {
         Unit = Literal | UniqueESSymbol | Nullable,
         StringOrNumberLiteral = StringLiteral | NumberLiteral,
         /* @internal */
-        StringOrNumberLiteralOrUnique = StringOrNumberLiteral | UniqueESSymbol,
+        StringOrNumberLiteralOrUnique = StringLiteral | NumberLiteral | UniqueESSymbol,

```

Comment:
this change doesn't seem related (and I'm not sure it's an improvement?)

Reply:
Eh, I changed ti because I actually removed `StringOrNumberLiteral` at first because it was practically unused within our code, but then added ti back to avoid an API break.